lib/screens/todo_overview_screen.dart

```diff
@@ -1,20 +1,83 @@
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../widgets/todo_items.dart';
+import '../models/constants.dart';
+import '../provider/todo_provider.dart';
+import '../widgets/add_todo.dart';
 
 class TodoOverviewScreen extends StatefulWidget {
- static const routeName = '/Todo-Overview';
+  static const routeName = '/todo-overview';
 
   @override
   _TodoOverviewScreenState createState() => _TodoOverviewScreenState();
 }
 
 class _TodoOverviewScreenState extends State<TodoOverviewScreen> {
+  final _todoController = TextEditingController();
```

Comment:
what do you need this for? in `TodoOverviewScreen`? Isn't this supposed to be in `AddTodo` Screen?